mxcube3/ui/serverIO.js

```diff
@@ -154,6 +159,11 @@ class ServerIO {
     });
 
     this.hwrSocket.on('sc', (record) => {
+      this.dispatch(setLoading((record.signal === 'operatingSampleChanger'),
+                               'Sample changer in operation',
+                               record.message, true, () => (this.dispatch(sendStopQueue()))));
```

Comment:
What does this signal signify ?, and are we certain that we like stopQueue as a abort action ?